apps/src/craft/code-connection/craft.js

```diff
@@ -218,7 +218,11 @@ export const executeUserCode = function (client, code) {
 
   // Register async methods
   codegen.asyncFunctionList = Object.values(asyncMethods);
-  interpreter = CustomMarshalingInterpreter.evalWith(code, Object.assign(asyncMethods, methods));
+  interpreter = CustomMarshalingInterpreter.evalWith(
+    code,
+    Object.assign(asyncMethods, methods),
+    {asyncFunctionList: asyncMethods}
```

Comment:
yep. That is a mistake! And probably the cause of the failing circle build. Will fix.